cpp/include/cudf/strings/findall.hpp

```diff
@@ -47,14 +48,16 @@ namespace strings {
  *
  * See the @ref md_regex "Regex Features" page for details on patterns supported by this API.
  *
- * @param strings Strings instance for this operation.
+ * @param input Strings instance for this operation.
  * @param pattern Regex pattern to match within each string.
+ * @param flags Regex flags for interpreting special characters in the pattern.
  * @param mr Device memory resource used to allocate the returned table's device memory.
  * @return New table of strings columns.
  */
 std::unique_ptr<table> findall(
-  strings_column_view const& strings,
+  strings_column_view const& input,
```

Comment:
I've come across some compiler bugs in the past that had trouble with using `strings` as a variable name inside a namespace called `strings`. These were painful to figure out since they generated bad code instead of giving a simple compile error. So I've been slowly changing the ones I find to `input` to avoid this. I also feels it makes it more clear what the variable is.

Reply:
Good question -- and thanks for the context @davidwendt, I would not have guessed that would happen.